### PR TITLE
[10.0][FIX] server_env_base_external_referentials. Make it pass mqt lint.

### PR DESCRIPTION
**Pull request by NL66278**
Part of an effort to make commits to server-env repo pass mqt pre-commit hook.

First try for this particular module was to solve the errors.

On closer inspection the module is bases on another module not maintained since 6.1.

Therefore deletion of the moduel seems to me the proper course of action.

### CODE REVIEW

File: server_env_base_external_referentials/__manifest__.py [hunk 1]
```diff
@@ -21,16 +21,20 @@
...
     'license': 'AGPL-3',
-    "description": """This module is based on the server_environment module to use files for configuration.
-Thus we can have a different file for each environment (dev, test, staging, prod).
-This module define the config variables for the base_external_referential module.
-In the configuration file, you can configure the url, login and password of the referentials
+    "description": """This module is based on the server_environment module
+to use files for configuration.
+Thus we can have a different file for each environment
+(dev, test, staging, prod).
+This module define the config variables for the base_external_referential
```
Comment (JayVora-SerpentCS) [inline server_env_base_external_referentials/__manifest__.py]: This module defines!

File: server_env_base_external_referentials/__manifest__.py [hunk 2]
```diff
@@ -21,16 +21,20 @@
...
+module.
+In the configuration file, you can configure the url, login and password
+of the referentials
```
Comment (JayVora-SerpentCS) [inline server_env_base_external_referentials/__manifest__.py]: DOT after referentials

File: server_env_base_external_referentials/base_external_referentials.py [hunk 3]
```diff
@@ -18,37 +18,40 @@
...
-                   method=True, string='Password', multi='connection_config'),
+        'location': fields.function(
+            _get_environment_config_by_name, type='char', size=200,
+            method=True, string='Location', multi='connection_config'),
+        'apiusername': fields.function(
+            _get_environment_config_by_name, type='char', size=64,
+            method=True, string='User Name', multi='connection_config'),
+        'apipass': fields.function(
+            _get_environment_config_by_name, type='char', size=64,
+            method=True, string='Password', multi='connection_config'),
```
Comment (JayVora-SerpentCS) [inline server_env_base_external_referentials/base_external_referentials.py]: method and size are not necessary params IMHO

**Comment by NL66278** [comment]
@JayVora-SerpentCS Thanks for looking into this. I considered doing the actual migration, but then found out it is based on a module that is no longer maintained since 6.1. Sorry for the work you also out in the first review, but I now think it is better to get rid of the whole thing.

**Comment by NL66278** [comment]
@JayVora-SerpentCS Do you agree with me that the module can better be removed? Then please approve in review.

**Review by JayVora-SerpentCS — APPROVED** [review]
Better to be removed yes

**Review by yvaucher — APPROVED** [review]
LGTM seems not ported in version 9.0

@guewen reading the manifest it seems you created it in relation to magento connector? Can we drop it?

AFAIK we have no more use of it in later versions.

**Pull request closed without merging**